test/Plutip/Utils.purs

```diff
@@ -10,31 +10,25 @@ import Contract.Monad (Contract)
 import Contract.Transaction
   ( BalancedSignedTransaction
   , awaitTxConfirmed
-  , getTxByHash
   , submit
   )
 import Control.Monad.Reader (asks)
 import Ctl.Internal.Types.UsedTxOuts (TxOutRefCache)
-import Data.Maybe (Maybe(Just), isNothing)
 import Data.Newtype (unwrap)
 import Effect.Class (liftEffect)
-import Effect.Exception (throw)
 import Effect.Ref as Ref
 
+-- TODO: Get everything we can about a tx and confirm them
```

Comment:
So this is functionality killed and waiting to be redone with new metadata function?

Reply:
Yeah it no longer verifies the transaction. I don't think it's a critical piece of code though. What exactly it should do isn't really clear by the name